### ISSUE TITLE
[Config Support]: motion keeps recording all

### ISSUE THREAD

**Issue by arahayrabedian** [blakeblackshear/frigate#7941]
### Describe the problem you are having

Hey, I've been battling with some configuration issues for some time now - I had it working at some point where frigate would only record `motion` correctly, however, after some changes (and version upgrades) I can no longer get it to work as expected. I'm pretty sure it's a mistake on my side but I've battled this for a while.

It worked earlier in the 0.13 dev branch work (though can't guarantee my config was exactly the same), and worked with 0.12 (though i wanted to set up audio detection without crashing and there were a few other bugs on dev, so trying to get it to work with the latest beta now)

In my attached config, i'm trying to get `littercam` to only record motion for 14 days. I have seen this work previously, but could not get it working again after like half a day's battle with the config.

Thanks so much for your hard work on the project!

### Version

0.13.0-0858859

### Frigate config file

```yaml
mqtt:
  enabled: False

detectors:
  coral:
    type: edgetpu
    device: usb

audio:
 enabled: True
 min_volume: 25
 listen:
   - bark
   - scream
   - yell
   - meow
   - growling
   - hiss
   - cat
   - door
   - doorbell
   - fart

objects:
  track:
    - person
    - cat
    - dog

record:
  enabled: True
  retain:
    days: 2
    mode: all
  events:
    retain:
      default: 2

go2rtc:
  rtsp:
    username: "user"
    password: "pass"
  streams:
    doggocam: 
      - rtsp://user:pass@doggocam.lan:554/s0
    littercam:
      - rtsp://user:pass@littercam.lan:554/s0

cameras:
  doggocam:
    ffmpeg:
      inputs:
        - path: rtsp://user:pass@127.0.0.1:8554/doggocam
          roles:
            - detect
            - record
            - audio
      output_args: 
        record: preset-record-generic-audio-aac
    detect:
      enabled: True
  littercam:
    ffmpeg:
      inputs:
        - path: rtsp://user:pass@127.0.0.1:8554/littercam
          roles:
            - detect
            - record
            - audio
      output_args: 
        record: preset-record-generic-audio-aac
    record:
      retain:
        days: 14  # <-- this is being applied
        mode: motion  # <-- this is not being applied
      events:
        retain:
          default: 14
    detect:
      enabled: True
    motion:
        mask:
        - 1347,891,1140,891,1348,1080,1920,1080,1920,0,1242,0
        - 1255,0,1243,88,618,48,622,0
        - 0,1080,432,1080,466,0,0,0
        # - 898,594,760,594,899,720,1280,720,1280,0,828,0
        # - 837,0,829,59,412,32,415,0
        # - 0,720,288,720,311,0,0,0
```

### Relevant log output

```shell
nothing too relevant
```

### Operating system

Other

### Install method

Docker Compose

### Coral version

USB

### Any other information that may be helpful

Basically, debug view for the camera (littercam in this case) shows no motion, however, it retains recordings for 14 days, so frigate *is* reading the camera-specific override block (otherwise it would only retain for 2 days)

Any tips on silly mistakes would be appreciated - is there any way to view frigate's internal state/parsing for this config?

Thanks!

**Comment by NickM-27** [comment]
ah, I see what is going on here. `motion` includes audio as well, if audio is detected with a volume > `audio -> min_volume` then that segment will be kept

**Comment by arahayrabedian** [comment]
oh wow, so even if it doesn't detect _something_ in the audio, just that it's crossing the threshold?

I can test that out.

**Comment by NickM-27** [comment]
yes, that way in the future frigate could show a visual (motion) and an audio heatmap on a timeline of when there was a lot of activity

**Comment by arahayrabedian** [comment]
preliminarily looks like that did the trick - i'll probably turn off global audio detect and just turn it on the one cam that actually needs it, thanks!

Nothing in the docs prepared me for this one unfortunately, but an understandable approach - perhaps a little "audio 'motion'" indicator would help in the debug view?

**Comment by arahayrabedian** [comment]
from my side, you can consider this closed I think, thank you for your prompt response!

**Comment by NickM-27** [comment]
This is covered in the audio docs https://deploy-preview-6262--frigate-docs.netlify.app/configuration/audio_detectors#configuring-minimum-volume

I think audio debugging could be cool, I'll go ahead and close this though for now

**Comment by arahayrabedian** [comment]
don't think that tip was there when I set it up originally some weeks back - one of the dangers of me jumping the gun with the feature :joy: 

Thanks again :)

**Comment by NickM-27** [comment]
ah yes, it was merged ~ a week ago
